tradable-app/src/pages/Dashboard/Dashboard.js

```diff
@@ -0,0 +1,422 @@
+                              color="secondary"
+                              style={{ fontSize: "0.8rem" }}
+                           >
+                              Average Daily Cost
+                           </Card.Subtitle>
+                        </Card.Body>
+                     </Card>
+                     <Card
+                        borderColor={globalTheme.colors.darkMode.light + "30"}
+                        width={Sizes.XS}
+                        style={{ margin: "0.5rem" }}
+                     >
+                        <Card.Body>
+                           <Card.Title color="light" size="m">
+                              $1,553
+                           </Card.Title>
+                           <Card.Subtitle
+                              color="secondary"
+                              style={{ fontSize: "0.8rem" }}
+                           >
+                              Average Daily Cost
+                           </Card.Subtitle>
+                        </Card.Body>
+                     </Card>
+                  </Card.LinksContainer>
+               </Card.Body>
+            </Card>
+            <TablesContainer>
+               <Card
+                  borderColor="transparent"
+                  style={{
+                     margin: "1rem",
+                     width: "100%",
+                     backgroundColor:
+                        globalTheme.colors.darkMode.cardBackground,
+                     borderRadius: "20px",
+                  }}
+               >
+                  <Card.Body>
+                     <Card.Title
+                        style={{
+                           color: globalTheme.colors.darkMode.light + "a0",
+                        }}
+                        color="light"
+                        size="s"
+                     >
+                        Top Services
+                     </Card.Title>
+                     <Table mode={getMode()} />
+                  </Card.Body>
+               </Card>
+               <Card
+                  borderColor="transparent"
+                  style={{
+                     margin: "1rem",
+                     width: "100%",
+                     backgroundColor:
+                        globalTheme.colors.darkMode.cardBackground,
+                     borderRadius: "20px",
+                  }}
+               >
+                  <Card.Body>
+                     <Card.Title
+                        style={{
+                           color: globalTheme.colors.darkMode.light + "a0",
+                        }}
+                        color="light"
+                        size="s"
+                     >
+                        Top Services
+                     </Card.Title>
+                     <Table mode={getMode()} />
+                  </Card.Body>
+               </Card>{" "}
+            </TablesContainer>
+         </Row>
+      </Container>
+   );
+};
+
+export default Dashboard;

```

Comment:
you should see how you cut the length of this component
looks to me that a loop will help printing the items